sdk/core/azure-core/inc/azure/core/http/win_http_transport.hpp

```diff
@@ -171,17 +171,13 @@ namespace Azure { namespace Core {
       /**
        * @brief Constructs `%WinHttpTransport` object based on common Azure HTTP Transport Options
        *
-       * @param options Common Azure Core Transport Options to override the default settings.
        */
       WinHttpTransport(Azure::Core::Http::Policies::TransportOptions const& options);
 
       /**
        * @brief Implements the HTTP transport interface to send an HTTP Request and produce an
        * HTTP RawResponse.
        *
-       * @param context A context to control the request lifetime.
-       * @param request an HTTP request to be send.
-       * @return A unique pointer to an HTTP RawResponse.
```

Comment:
Why do we have to remove these? Is it because of the function being virtual?

Reply:
Doxygen gets annoyed when the documentation for a function appears in multiple locations. This applies when the function is declared, it also applies when a virtual function overload is defined.

Reply:
And ultimately, we had to remove them because doxygen complained about their existance.